test/LaTeXAndHTML/succeed/Issue2474.tex

```diff
@@ -29,6 +29,6 @@
 \\
 \>[0]\AgdaFunction{A}\AgdaSpace{}%
 \AgdaSymbol{=}\AgdaSpace{}%
-\AgdaUnsolvedMeta{\AgdaPostulate{F}}\<%
+\AgdaPostulate{F}\<%
```

Comment:
So this PR unfortunately unfixes #2474.
The unsolved meta location does not correspond to tokens anymore, so there is no place where to put the `\AgdaUnsolvedMeta` aspect.